components/style/properties.mako.rs

```diff
@@ -5724,6 +5724,97 @@ pub struct PropertyDeclarationBlock {
     pub normal: Arc<Vec<PropertyDeclaration>>,
 }
 
+impl PropertyDeclarationBlock {
+    // https://drafts.csswg.org/cssom/#serialize-a-css-declaration-block
```

Comment:
It'd be nice if you can put in some annotations about which step of the algorithm that the code is following. Just a `// Step 1.` above step 1 would be enough, for example.